tests/playwright/sfcc.config.js

```diff
@@ -0,0 +1,128 @@
+// @ts-check
+const { devices } = require('@playwright/test');
+
+/**
+ * @see https://playwright.dev/docs/test-configuration
+ * @type {import('@playwright/test').PlaywrightTestConfig}
+ */
+const config = {
+  testDir: './fixtures/',
+  testMatch: '*.spec.mjs',
+
+  /* Maximum time one test can run for. */
+  timeout: 90 * 1000,
+
+  expect: {
+    /**
+     * Maximum time expect() should wait for the condition to be met.
+     * For example in `await expect(locator).toHaveText();`
+     */
+    timeout: 10000,
+  },
+
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 2 : undefined,
+
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: 'html',
+
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Maximum time each action such as `click()` can take. Defaults to 0 (no limit). */
+    actionTimeout: 20000,
+
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    baseURL: process.env.SFCC_HOSTNAME,
+    httpCredentials: {
+      username: process.env.SANDBOX_HTTP_AUTH_USERNAME,
+      password: process.env.SANDBOX_HTTP_AUTH_PASSWORD,
+    },
+
+    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+    trace: 'retain-on-failure',
+  },
+
+  /* Configure projects for major browsers */
+  projects: [
+    {
+      name: 'chromium',
+
+      /* Project-specific settings. */
+      use: {
+        ...devices['Desktop Chrome'],
+      },
+    },
+
+    {
+      name: 'firefox',
+      use: {
+        ...devices['Desktop Firefox'],
+      },
+    },
+
+    {
+      name: 'webkit',
+      use: {
+        ...devices['Desktop Safari'],
+      },
+    },
+
+    {
+      name: 'chrome',
+      use: {
+        channel: 'chrome',
+      },
+    },
+
+    {
+      name: 'edge',
+      use: {
+        channel: 'msedge',
+      },
+    },
+
+    /* Test against mobile viewports. */
```

Comment:
do we need to keep this commented code?

Reply:
It's a default config file, would be nice to keep them imho if we ever need to enable test cases for mobile viewports.